src/tools/rest-frontend/README.md

```diff
@@ -6,15 +6,15 @@ This document aims to provide information about Elektras `rest-frontend`, which
 
 ## Design and Structure
 
-The frontend is developed as Single Page Application in [AngularJS (v1.5)](https://angularjs.org/). All dependencies are either already contained in the application project or (preferred) resolved through [npm](https://www.npmjs.com/) and [bower](https://bower.io/) during installation (requires active internet connection). Compiling (concatenation & minification), as well as other tasks like running a lightweight webserver are handled by the task runner `grunt`.
+The frontend is developed as Single Page Application in [AngularJS (v1.5)](https://angularjs.org/). All dependencies are either already contained in the application project or (preferred) resolved through [npm](https://www.npmjs.com/) during installation (requires active internet connection). Compiling (concatenation & minification), as well as other tasks like running a lightweight webserver are handled by the task runner `grunt`.
```

Comment:
Angular 2 has completely different syntax and design compared to Angular 1. At the time when I started working on the frontend, there was no stable release of Angular 2 available yet (release was mid september). Migration isn't easy in this case, also because most of the plugins/modules I use are not (yet) available for Angular 2.

I'll change the text, it needs some addition now anyway.